src/js/utils.js

```diff
@@ -16,6 +16,27 @@ exports.is_sample_genetically_altered = function is_sample_genetically_altered(d
   || datum.protein !== undefined;
 };
 
+exports.makeD3SVGElement = function(tag) {
```

Comment:
As usual, it's shuffling data around one way or another way. It's shuffling around the parameters to create an object versus passing around the object itself (I am talking about the DOM object). I would say that it's best to be consistent and not introduce special data types (DOM objects) for non-special cases, but it's your call.